locales/fr-FR.json

```diff
@@ -78,7 +78,7 @@
         "2": "Serrez à <RELATIVE_DIRECTION> dans <BEGIN_STREET_NAMES>.",
         "3": "Serrez à <RELATIVE_DIRECTION> pour rester sur <STREET_NAMES>.",
         "4": "Bear <RELATIVE_DIRECTION> at <JUNCTION_NAME>.",
-        "5": "Bear <RELATIVE_DIRECTION> toward <TOWARD_SIGN>."
+        "5": "Serrez à <RELATIVE_DIRECTION>vers<TOWARD_SIGN>."
```

Comment:
Should there be a space on each side of vers

Reply:
Good catch, fixed.